### ISSUE TITLE
Improving House

### ISSUE THREAD

**Issue by H4oK3** [bad-hombres/derpsuite#1]
Hey, sorry for reaching you via this method;

I am the developer of https://github.com/nccgroup/house, and thanks for extending my work. I came across your repo yesterday and really great job! 
I was thinking about adding the auto-complete functionalities into house but I saw you already implemented here! I was wondering if we can work together a little bit and extend House a little bit more with the nice features you have it here. I think we could contribute more to open-source project society if we collaborate together, let me know if you are interested in helping out!

**Comment by bad-hombres** [comment]
@H4oK3 yeah sure! I was inspired by House and wanted to learn about frida so decided to try and build something with similar features to Burp but for android apps. The code is not great at the moment as it was a couple of days prototype. Let me know what you're thinking of!

**Comment by H4oK3** [comment]
Hey, great work, I am happy my tool can help you a little bit with Frida too; if you feel like improving house a little bit that would be great; right now I am thinking about add this enhancement to  House: https://github.com/nccgroup/house/issues/4; it would be super handy to get the auto-complete in each user input field that you have already implemented here, so I was thinking it would be great if you can port the code a little bit and submit a PF :)
